csxtools/lmfit_models.py

```diff
@@ -0,0 +1,200 @@
+import numpy as np

```

Comment:
I didn't see any conflicts with scikit-beam fitting functions. In scikit-beam, we mainly focus on 1D fitting, while what implemented here are about 2D curve fitting. It will be good to add those functions to scikit-beam eventually.